django_loci/base/admin.py

```diff
@@ -69,22 +69,22 @@ def get_urls(self):
         # without having to change templates
         app_label = 'django_loci'
         return [
-            re_path(
+            path(
                 r'^(?P<pk>[^/]+)/json/$',
                 self.admin_site.admin_view(self.json_view),
                 name='{0}_location_json'.format(app_label),
             ),
-            re_path(
+            path(
                 r'^(?P<pk>[^/]+)/floorplans/json/$',
                 self.admin_site.admin_view(self.floorplans_json_view),
                 name='{0}_location_floorplans_json'.format(app_label),
             ),
-            re_path(
+            path(
                 r'^geocode/$',
```

Comment:
Hi, @queenish001 I guess we don't require, the url pattern to be in regex format for simple patterns when using `path`, can you look at it once. :smile:

Reply:
if these works, do we really need to change all of them in all modules?
@pandafy @atb00ker @codesankalp what do you think?

Reply:
We can use `re_path` instead of changing all the regex-based URL patterns in all modules.

Reply:
Um, So should I :
1. Change it back to `re_path`
Or
2. Keep it `path` and work on that regex issue?

Reply:
let's do like it was done in openwisp-users and as Manish suggested initially, which looks cleaner and better for the long run: https://github.com/openwisp/openwisp-users/commit/289801e629cf969e5ea839dbf360c0914d443d01

Please do alwaysy some manual testing though, @queenish001, I ran the app and tried to open the admin page and I am getting 404, which hints to me you did the application after your changes which is a huge red flag and should never be done, always test the software you work on, not only with automated test suite but also with manual testing, and if how to do manual testing is not clear please ask.